ext/openssl/tests/openssl_pkcs7_read.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
```

Comment:
might be good to rename to `openssl_pkcs7_read_basic.phpt` to keep it consistent